package.json

```diff
@@ -3,6 +3,7 @@
   "version": "2.7.0",
   "description": "Collection of Resin.io JavaScript errors",
   "main": "build/errors.js",
+  "types": "build/errors.d.ts",
```

Comment:
Really? Is there a good reason for that? I think our standard approach everywhere else is to publish the lib with the TS, point `types` at that directly, and skip generating type definitions. I don't know of any downsides, and it's definitely simpler.